src/pages/DetailGrade/index.js

```diff
@@ -0,0 +1,264 @@
+    'point',
+    'User.picture',
+  ]
+  let keymapUser = {
+    'User.id': 'id',
+    'User.username': 'username',
+    'User.firstName': 'firstName',
+    'User.lastName': 'lastName',
+    'User.studentId': 'studentId',
+    'User.picture': 'picture',
+  }
+  useEffect(() => {
+    const getGradeDetail = async () => {
+      try {
+        const res = await axiosClient.get(`/api/classrooms/${id}/grades`)
+        // Format user array
+        res.data.forEach((grade, index) => {
+          res.data[index].users = grade.users.map((user) =>
+            _.pick(user, mapUser)
+          )
+        })
+        //--------------------
+        // Get all students in classroom
+        const response = await axiosClient.get(`/api/classrooms/${id}/users`)
+        const users = response.data.filter((user) => user.role === 'STUDENT')
+        const listGradeStudent = res.data
+        if (users && listGradeStudent) {
+          let arrData = []
+          let tempCol = []
+          //Add field to column
+          listGradeStudent.forEach((g) => {
+            const temp = {
+              field: g.name.split(' ').join(''),
+              headerName: g.name,
+              width: 150,
+            }
+            tempCol.push(temp)
+          })
+          //--------------
+          users.forEach((user) => {
+            let row = [
+              _.mapKeys(_.pick(user, mapUser), function (v, k) {
+                return keymapUser[k]
+              }),
+            ]
+            listGradeStudent.forEach((g) => {
+              const userFilter = g.users.filter(
+                (u) => lodashGet(u, 'User.id') === user.userId
+              )
+              //Check student exist in array user
+              if (userFilter.length > 0) {
+                if (userFilter[0].point) {
+                  row[0][g.name.split(' ').join('')] = userFilter[0].point
+                } else {
+                  row[0][g.name.split(' ').join('')] = null
+                }
+              }
+            })
+            arrData.push(row[0])
+          })
+          setRows(arrData)
+          const col = columns.concat(tempCol)
+          setColumns(col)
+        }
+        setListGradeStudent(listGradeStudent)
+        setUsers(response.data.filter((user) => user.role === 'STUDENT'))
+      } catch (error) {
+        console.log('Grade Detail Error', error)
+      }
+    }
+    getGradeDetail()
+  }, [])
+  console.log('User', rows)
+  return (
+    <Layout>
+      <ThemeProvider theme={theme}>
+        <CssBaseline />
+        <Box
+          sx={{
+            marginTop: 2,
+            display: 'flex',
+            flexDirection: 'column',
+            alignItems: 'center',
+          }}
+        >
+          <CustomCard sx={{ minWidth: 1200 }} className={`sticky`}>
+            <CardHeader
+              avatar={
+                <IconButton onClick={goBack}>
+                  <ArrowBackIosNew />
+                </IconButton>
+              }
```

Comment:
I get confuse at this code. Why CardHeader have avatar props and that avatar props is a back button 🤣

Reply:
Pls ignore it,  because according to CardHearder API, avatar props is a good spot for the goBack button.